python/cudf/cudf/tests/test_groupby.py

```diff
@@ -1120,3 +1120,18 @@ def test_reset_index_after_empty_groupby():
         pdf.groupby("a").sum().reset_index(),
         gdf.groupby("a").sum().reset_index(),
     )
+
+
+def test_groupby_attribute_error():
+    err_msg = "Test error message"
+
+    class TestGroupBy(cudf.core.groupby.GroupBy):
```

Comment:
We're depending on some internals of groupby in this test. I don't like it but couldn't figure out a better solution

Reply:
By internals are you just referring to `cudf.core.groupby.GroupBy`?